docs/source/backends.rst

```diff
@@ -0,0 +1,109 @@
+.. role:: hidden
+    :class: hidden-section
+
+torch.backends
+==============
+
+`torch.backends` controls the behavior of various backends that PyTorch supports.
+
+These backends include:
+
+- ``torch.backends.cuda``
+- ``torch.backends.cudnn``
+- ``torch.backends.mkl``
+- ``torch.backends.mkldnn``
+- ``torch.backends.openmp``
+- ``torch.backends.quantized``
+- ``torch.backends.xnnpack``
+
+
+torch.backends.cuda
+^^^^^^^^^^^^^^^^^^^
+
+.. autofunction::  torch.backends.cuda.is_built
+
+.. attribute::  torch.backends.cuda.matmul.allow_tf32
+
+    A :class:`bool` that controls whether TensorFloat-32 tensor cores are allowed for the
+    computation of matrix multiplications. TensorFloat-32 tensor cores is a new technology
+    available on Ampere or newer GPUs that allows speeding up certain computations at the
+    cost of precision.
+
+    See :ref:`tf32_on_ampere` for more information.
```

Comment:
Will this display the hidden link tag `tf32_on_ampere` directly on the rendered page, or will it display the full title of the target (`TensorFloat-32(TF32) on Ampere devices`)?  If it displays the full target title I think that's alright.

If you want to control the displayed text to something shorter, this is the syntax, maybe you already know.
```
:ref:`Displayed Text<tf32_on_ampere>`
```

Reply:
It is the full title:
![image](https://user-images.githubusercontent.com/1032377/93241282-ff800d00-f739-11ea-8f61-f83e536cb301.png)

But thank you for letting me know the `:ref:`Displayed Text<tf32_on_ampere>`, I didn't know there is a way to specify the text.